lo pude resolver

guillelizard opened the pull request:
perdón la demora me costo un poco el manejo de github pero lo logre

guillelizard committed:
lo pude resolver

perdón la demora me costo un poco el manejo de github pero lo logre

enzzo19 commented:
Hola Guille muy bien, que bueno que te animaste :), mira aboserve algunas cosas en tu solucion:
* La funcion que invocas y la que defines tienen un nombre diferente por un letra, la ultima.
* El parametro con el que invocas la funcion `frase` no esta definido como una variable global, entonces aun no esta definido a la hora de probarlo.

Intenta corregir esto y lo seguimos trabajando, si necesitas ayuda me avisas 👍

guillelizard committed:
resuelto,con las correcciones

hola,perdón la demora,espero estén bien las correcciones que me sugirió que hice. para la próxima voy a mejorar jejeje. saludos
